app/helpers/decidim/menu_helper.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Decidim
+  # Override helper to manage menus in layout
+  module MenuHelper
```

Comment:
This customization is missing the `user_menu` method.

Now if I sign in and click the top bar user menu -> select "My account" -> book:
![User menu](https://user-images.githubusercontent.com/864340/194573102-e38301e3-f796-4494-b1b5-d64cccbbbe97.png)

Could you still check that?
https://github.com/decidim/decidim/blob/fcac16c9eed35c9ca0ac30a3be7b962881aa06ee/decidim-core/app/helpers/decidim/menu_helper.rb#L18-L26

Reply:
Done in b1a19ad65bd7256d34bd6861f6725bacd2711ce4 
I also added a spec just in case